seed/views/v3/users.py

```diff
@@ -209,64 +138,51 @@ def validate_request_user(self, pk, request):
                 status=status.HTTP_403_FORBIDDEN)
         return True, user
 
+    @swagger_auto_schema(
+        manual_parameters=[AutoSchemaHelper.query_org_id_field()],
+        request_body=openapi.Schema(
```

Comment:
Sorry, I should've been more descriptive - I meant adding this back via a description decorator attribute
```
description="An object containing meta data for a new user: \n"
                                "- Required - first_name, last_name, email \n"
                                "- Optional - role ['viewer'(default), 'member', or 'owner']",
```

But actually, the `required` openapi.Schema attribute is a good substitute to that documentation.